.processes/wallets.md

```diff
@@ -0,0 +1,35 @@
+# Wallets
+
+At HOPR we create and use multiple Ethereum addresses that are under the control and supervision of the HOPR Association.
+
+Here are some of these wallets:
+
+- `0xA18732DC751BE0dB04157eb92C92BA9d0fC09FC5` (codename: Deployer)
+- `0x1A387b5103f28bc6601d085A3dDC878dEE631A56` (codename: Minter)
+- `0x34465FE0B25089Fc9D3a6D33e19F652e45B175e0` (codename: Alice)
```

Comment:
maybe turn into a table and add at least a minimal explanation of the usage. e.g. `Alice` is not self-explanatory.

Reply:
Resolved in 8e439fb